gpu/src/structs.zig

```diff
@@ -39,9 +39,9 @@ pub const PrimitiveState = struct {
 
 pub const StorageTextureBindingLayout = extern struct {
     reserved: ?*anyopaque = null,
-    access: StorageTextureAccess,
-    format: Texture.Format,
-    view_dimension: TextureView.Dimension,
+    access: StorageTextureAccess = .none,
+    format: Texture.Format = .none,
```

Comment:
The spec has this to say - I think we should make this required for now, no default `.none`:

<img width="507" alt="image" src="https://user-images.githubusercontent.com/3173176/162637605-b3c65ddd-be43-4fdc-b7f1-a963b484d2e2.png">